content/github/setting-up-and-managing-your-github-profile/managing-your-profile-readme.md

```diff
@@ -54,6 +54,8 @@ GitHub will display your profile README on your profile page if all of the follo
   The generated README file is pre-populated with a template to give you some inspiration for your profile README.
   ![README file with pre-populated template](/assets/images/help/repository/personal-repository-readme-template.png)
 
+**Note**: If you want to add emojis to your README, you can find a list of the required code and accompanying output [here](https://github.com/ikatyang/emoji-cheat-sheet/blob/master/README.md).
```

Comment:
Hi David - thanks for this suggestion. You're right, this looks like a good place to add a link to an emoji reference. I also like the link you've chosen, but we should probably use the same cheat sheet here as we do in other topics 
(for example, [Using emoji](https://docs-2381--patch-1.herokuapp.com/en/free-pro-team@latest/github/writing-on-github/basic-writing-and-formatting-syntax#using-emoji)).

My suggestion below also reorganizes this sentence so that it's easier for the localization teams.

```suggestion
For a summary of all the available emojis and their codes, see "[Emoji cheat sheet](http://www.emoji-cheat-sheet.com/)."
```